ui/job-view/details/tabs/failureSummary/SuggestionsListItem.jsx

```diff
@@ -40,56 +110,13 @@ export default class SuggestionsListItem extends React.Component {
           </Button>
           <span>{suggestion.search}</span>
         </div>
-
-        {/* <!--Open recent bugs--> */}
-        {suggestion.valid_open_recent && (
-          <ul className="list-unstyled failure-summary-bugs">
-            {suggestion.bugs.open_recent.map((bug) => (
-              <BugListItem
-                key={bug.id}
-                bug={bug}
-                suggestion={suggestion}
-                selectedJobFull={selectedJobFull}
-              />
-            ))}
-          </ul>
-        )}
-
-        {/* <!--All other bugs--> */}
-        {suggestion.valid_all_others && suggestion.valid_open_recent && (
-          <Button
-            color="link"
-            rel="noopener"
-            onClick={this.clickShowMore}
-            className="show-hide-more"
-          >
-            Show / Hide more
-          </Button>
-        )}
-
-        {suggestion.valid_all_others &&
-          (suggestionShowMore || !suggestion.valid_open_recent) && (
-            <ul className="list-unstyled failure-summary-bugs">
-              {suggestion.bugs.all_others.map((bug) => (
-                <BugListItem
-                  key={bug.id}
-                  bug={bug}
-                  suggestion={suggestion}
-                  bugClassName={bug.resolution !== '' ? 'strike-through' : ''}
-                  title={bug.resolution !== '' ? bug.resolution : ''}
-                  selectedJobFull={selectedJobFull}
-                />
-              ))}
-            </ul>
-          )}
-
-        {(suggestion.bugs.too_many_open_recent ||
-          (suggestion.bugs.too_many_all_others &&
-            !suggestion.valid_open_recent)) && (
-          <mark>
-            Exceeded max {thBugSuggestionLimit} bug suggestions, most of which
-            are likely false positives.
-          </mark>
+        {suggestions.length > 0 && (
```

Comment:
So here is the biggest reason for this refactor. In order to create the vertical line with the indentation, I have a new container div to control the padding. In addition, I've added an `h4` title here.

<img width="977" alt="image" src="https://user-images.githubusercontent.com/1588648/83075926-44bce980-a03a-11ea-8b2b-5d7fb3ea6c57.png">

So I have to do 2 things here, I have to increase the indentation of all of the items, and I have to be able to not include the the container if it would be empty of suggestions. This is why I've collected everything on a `suggestions` array in order to make this check.

Reply:
Also, for the readability, originally I split the move refactor into a separate commit from the behavior changes.

The behavior changes was just: https://github.com/mozilla/treeherder/pull/5136/commits/22bbc34bfebad1d61bf91551a6230e6d11dcface

I've added 2 more commits at this point for addressing review feedback. Let me know what you'd like me to do, but I don't think think I'll be able to achieve the style changes without this refactor, which I did attempt.

Reply:
Thanks for explaining your thought process. I think these changes look OK.